src/client/src/components/Header.js

```diff
@@ -2,19 +2,35 @@ import  React from "react";
 import { Link as RouterLink } from "react-router-dom";
 import {AppBar, Button, Toolbar, Typography } from "@material-ui/core";
 import styles from "./styles/header.module.css";
+import admstyles from "./styles/adm_header.module.css";
```

Comment:
it's ok for now, but we are trying to avoid css files and work with the withStyles/makeStyles, we can refactor later

Reply:
ahh got it. we will need to change that in the fututre, but different scope